yt_dlp/extractor/playsuisse.py

```diff
@@ -0,0 +1,296 @@
+                'description': 'md5:8ea7a8076ba000cd9e8bc132fd0afdd8'
+            }
+        },
+        {
+            'url': 'https://www.playsuisse.ch/watch/808675/0',
+            'md5': '7aa043e69fea5044db2da8bb58bca239',
+            'info_dict': {
+                'id': '808675',
+                'ext': 'mp4',
+                'title': 'Der Läufer',
+                'description': 'md5:'
+            }
+        },
+        {
+            'url': 'https://www.playsuisse.ch/watch/817913/0',
+            'md5': '50721c46ca0b3a9836eb61ecb0ed7097',
+            'info_dict': {
+                'id': '42',
+                'ext': 'mp4',
+                'title': 'Nr. 47 S01E01 - Die Einweihungsparty',
+                'description': 'md5:'
+            }
+        }
+    ]
+
+    def _get_media_data(self, media_id):
+        locale = std_headers.get('locale', 'de').strip()
+        # TODO find out why the locale has no effect in request
+        response = self._download_json(
+            self._MEDIA_URL,
+            media_id,
+            data=json.dumps({
+                'operationName': 'AssetWatch',
+                'query': self._GRAPHQL_QUERY,
+                'variables': {
+                    "assetId": media_id
+                }
+            }).encode('utf-8'),
+            headers={'Content-Type': 'application/json', 'locale': locale})
+
+        return response['data']['asset']
+
+    def _real_extract(self, url):
+        media_id, = self._VALID_URL_RE.match(url).groups()
+        media_data = self._get_media_data(media_id)
+
+        def _get_title(data):
+            if data.get('seriesName'):
+                return '{} S{:02}E{:02} - {}'.format(
+                    data['seriesName'],
+                    int(data['seasonNumber']),
+                    int(data['episodeNumber']),
+                    data['name'])
+            else:
+                return data['name']
+
+        title = _get_title(media_data)
+        description = media_data['description']
+        thumbnails = [
+            {
+                'id': thumb['id'],
+                'url': thumb['url']
+            }
+            for key, thumb in media_data.items()
+            if key.startswith('thumbnail') and thumb is not None
+        ]
+
+        for media in media_data['medias']:
+            if media['type'] == 'HLS':
+                formats, subtitles = self._extract_m3u8_formats_and_subtitles(
+                    media['url'],
+                    media_id,
+                    'mp4',
+                    'm3u8_native',
+                    m3u8_id="HLS",
+                    fatal=False)
+
+            # NOTE seems to be 404 for all tested media
+            elif media['type'] == 'DASH' and False:
+                formats.extend(self._extract_mpd_formats(
+                    media['url'],
+                    media_id,
+                    mpd_id='dash',
+                    fatal=False
+                ))
```

Comment:
```suggestion
        formats, subtitles = [], {}
        for media in media_data['medias']:
            if media.get('type') == 'HLS':
                f, subs = self._extract_m3u8_formats_and_subtitles(
                    media['url'], media_id, 'mp4', 'm3u8_native', m3u8_id='HLS', fatal=False)
            elif media.get('type') == 'DASH':
                f, subs = self._extract_mpd_formats_and_subtitles(
                    media['url'], media_id, mpd_id='dash', fatal=False))
            else:
                continue
            formats.append(f)
            self._merge_subtitles(subtitles, subs)
```

OR

```suggestion
        formats, subtitles = [], {}
        for media in media_data['medias']:
            if media.get('type') != 'HLS':
                continue
            f, subs = self._extract_m3u8_formats_and_subtitles(
                media['url'], media_id, 'mp4', 'm3u8_native', m3u8_id='HLS', fatal=False)
            formats.append(f)
            self._merge_subtitles(subtitles, subs)
```

depending on what the deal is with DASH

Reply:
```
self._merge_subtitles(subs, target=subtitles)
```

Reply:
in this case, `self._merge_subtitles(subtitles, subs)` and `self._merge_subtitles(subs, target=subtitles)` are the same, no?

Reply:
https://github.com/yt-dlp/yt-dlp/blob/f1f6ca78b439343aa3f8ef44f803befd682a3d37/yt_dlp/extractor/common.py#L3483-L3491